actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -173,11 +173,14 @@ def _back_url # :nodoc:
       #   link_to "Visit Other Site", "http://www.rubyonrails.org/", data: { confirm: "Are you sure?" }
       #   # => <a href="http://www.rubyonrails.org/" data-confirm="Are you sure?">Visit Other Site</a>
       #
-      # Also you can set target attribute with <tt>target</tt> options:
+      # Also you can set any link attributes such as <tt>target</tt>, <tt>rel</tt>, <tt>type</tt>
+      # using these HTML options:
```

Comment:
Actually let's just :scissors: this line and put the colon at the end of the previous line :smile: